scripts/create_csv.py

```diff
@@ -121,7 +122,6 @@
 B = SimpleNamespace(**B)
 
 DEFAULTS = {
-    "parentPolicies": "sb-phishing-page-2,sb-infected-page-2",
```

Comment:
I notice that you removed this default, but you did not replace it with another default. So this means there is no default setting for the policies, correct? This is not a problem, I just wanted to confirm that it was intentional. Thanks.

Reply:
Yes, the latest version has no defaults for either the `policies` or `parentPolicies` parameters. The previous version forced the targeting of the `parent_policies` column, but this one allows for either `policies` or `parent_policies` to be targeted. The assumption is that **exactly one** of the `policies` or `parentPolicies` params will be set depending on which column should be targeted; if both are set, the job will actually refuse to run. The job knows which column to target based on which param actually received a value, i.e., setting `policies` targets the `policies` field, while setting `parentPolicies` targets the `parent_policies` field. Since giving a default for both parameters results in both params having a non-null value, the outcome is that the job refuses to run. We can write around this by basically having one of the params have priority over the other such that if both are set, only the preferred column type will be targeted, but I didn't want to make assumptions about which column should get that priority. As a possible side benefit, getting rid of the defaults forces the job configuration to be very clear about whether the `policies` or `parent_policies` column is being targeted. This is arguably a good thing considering how easy it is to either a) confuse the two very similarly named fields, or b) overlook the fact that the option to select a specific field exists at all.